themes/backendbrasil/assets/styles/menu.scss

```diff
@@ -1,9 +1,11 @@
 .be--navbar {
   @extend %has-default-padding;
-  background-color: transparent;
   padding-top: 0;
   padding-bottom: 0;
   width: 100%;
+  top: 0;
+  left: 0;
+  border-bottom: 3px solid #868686;
```

Comment:
essa borda não precisa, porque a cor vai ficar padrão da pagina atual e pode atrapalhar onde o fundo não for branco